.travis.yml

```diff
@@ -25,7 +25,6 @@ before_install:
 sudo: required
 stages:
 - install
-- test
```

Comment:
@limfriend, please rebase with upstream/master. The removal of the `test` stage was temporary -- it has been added back.